src/app/profile/yourGiving/stopStartRecurringGifts/restartGift/step1/suspendedGifts/suspendedGifts.tpl.html

```diff
@@ -0,0 +1,57 @@
+<div class="modal-header">
+  <div class="container-fluid">
+    <div class="row">
+      <div class="col-xs-12">
+        <div class="border-bottom-small">
+          <button type="button" class="close" ng-click="$ctrl.cancel()" aria-label="{{'Close' | translate}}">
+            <span aria-hidden="true">×</span>
+          </button>
+          <h3 translate>Restart A Gift</h3>
+          <p translate>You paused your giving to these recipients. Which gifts would you like to restart? You will be able to make edits to these gifts on the next step.</p>
+        </div>
+      </div>
+    </div>
+  </div>
+</div>
+
+<div class="modal-body">
+  <div class="container-fluid">
+    <div class="row row-no-spacing">
+      <div class="col-xs-12">
+        <form name="$ctrl.restartForm">
+          <div class="repeating-row select-recipient-row" ng-repeat="gift in $ctrl.gifts">
+            <gift-list-item gift="gift"
+                            selectable="checkbox">
+              <label>{{'Restart' | translate}}
+                <input type="checkbox" disabled>
+              </label>
+              <p translate>This gift cannot be changed online. For assistance,
```

Comment:
It's complicated. Step 1 can show up to 3 different modals to allow the user to restart a gift. The initial modal shows suspended gifts, which you can't restart in give currently (backend post deploy feature). If the user has not suspended gifts, you skip this modal. We then show suggested recipients, You can pick 1 or more of these to start giving gifts to (From user perspective: "I sent in a check twice last year, and now I want to 'restart' my giving online). If you have no suggested recipients, you skip it. If you have no suspended or suggested, or the user didn't select anything here, we show the Search Box. User is allowed to pick 1 (radio input) here. It then proceeds to configuration and confirmation.

I wasn't able to find a user with suspended gifts, so in development, it was always skipping to suggested.